### PR TITLE
Update contributing file

### PR DESCRIPTION
**Pull request by okaycj**
Updating contribution instructions to reflect current local development schemes. 

WIP:  I need to put this particular ball down from a moment.  I will double back on Monday and double check the changes.  Feel free to comment if you see some egregious errors.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -166,117 +161,14 @@ Django apps deliver translated strings. In summary:
...
-          information about the evolution of the product. Ideally, you'll be able to enter the commit SHA for the
-          updated version of [Lookit Orchestrator][lookit-orchestrator] that will be controlling your re-deployment.
-2. Now you're in a new release branch with the changed `VERSION` file in your git staging area (uncommitted). You should
-   make any changes you need to. Ideally, this is only additions to `CHANGELOG.md`.
-3. `git flow release finish $(cat VERSION)` - this will execute a `--no-ff` merge on both develop and master branches,
-   along with creating the appropriate tags both locally and remotely.
-
 ## Contact
+
 Rico Rodriguez, Lead Software Engineer ([rrodrigu@mit.edu](mailto:rrodrigu@mit.edu))
```
Comment (kimberscott) [inline CONTRIBUTING.md:172]: go ahead and list yourself! :)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -166,117 +161,14 @@ Django apps deliver translated strings. In summary:
...
 
 Kim Scott, Lookit Project Head ([kimscott@mit.edu](mailto:kimscott@mit.edu))
```
Comment (kimberscott) [inline CONTRIBUTING.md:174]: and we can change this too shortly.

**Comment by mekline** [comment]
@okaycj can you please review & resolve?

**Comment by okaycj** [comment]
> @okaycj can you please review & resolve?

I am going to need time with this file.  I know is seems like a small thing, but I have to make some decisions with this file.  I'll circle back to this tomorrow.  But, yes, I will review and resolve it shortly.

**Comment by mekline** [comment]
I think we probably want to have at least a stub CONTRIBUTING.md file but let's discuss at standup this afternoon!

**Comment by okaycj** [comment]
This file is updated in another PR.  Closing this PR.

**Pull request closed without merging**